great_expectations/render/renderer/column_section_renderer.py

```diff
@@ -107,62 +200,248 @@ def _render_overview_table(cls, evrs, content_blocks):
+            new_content_block["header"] = "Properties"
+            new_content_block["styling"] = {
+                "classes": ["col-4", ],
+                "styles": {
+                    "margin-top": "20px"
+                },
+                "body": {
+                    "classes": ["table", "table-sm", "table-unbordered"],
+                    "styles": {
+                        "width": "100%"
+                    },
+                }
+
+            }
+            content_blocks.append(new_content_block)
 
     @classmethod
     def _render_stats_table(cls, evrs, content_blocks):
-        min_evr = cls._find_evr_by_type(
-            evrs,
-            "expect_column_min_to_be_between"
-        )
+        table_rows = []
+
         mean_evr = cls._find_evr_by_type(
             evrs,
             "expect_column_mean_to_be_between"
         )
+        mean_value = "{:.2f}".format(
+            mean_evr['result']['observed_value']) if mean_evr else None
+        if mean_value:
+            table_rows.append(["Mean", mean_value])
+
+        min_evr = cls._find_evr_by_type(
+            evrs,
+            "expect_column_min_to_be_between"
+        )
+        min_value = "{:.2f}".format(
+            min_evr['result']['observed_value']) if min_evr else None
+        if min_value:
+            table_rows.append(["Minimum", min_value])
+
         max_evr = cls._find_evr_by_type(
             evrs,
             "expect_column_max_to_be_between"
         )
-        evrs = [evr for evr in [min_evr, mean_evr, max_evr] if evr is not None]
-        if len(evrs) > 0:
-            content_blocks.append(
-                TableContentBlockRenderer.render(evrs)
-            )
+        max_value = "{:.2f}".format(
+            max_evr['result']['observed_value']) if max_evr else None
+        if max_value:
+            table_rows.append(["Maximum", max_value])
+
+        if len(table_rows) > 0:
+            content_blocks.append({
+                "content_block_type": "table",
+                "header": "Statistics",
+                "table_rows": table_rows,
+                "styling": {
+                    "classes": ["col-4"],
+                    "styles": {
+                        "margin-top": "20px"
+                    },
+                    "body": {
+                        "classes": ["table", "table-sm", "table-unbordered"],
+                    }
+                },
+            })
+        else:
+            return
 
     @classmethod
     def _render_values_set(cls, evrs, content_blocks):
-        # FIXME: Eugene, here's where the logic for using "cardinality" in form of which expectations are defined
-        # should determine which blocks are generated
-        # Relatedly, this will change to grab values_list and to use expect_column_distinct_values_to_be_in_set
         set_evr = cls._find_evr_by_type(
             evrs,
-            "expect_column_distinct_values_to_be_in_set"
+            "expect_column_values_to_be_in_set"
```

Comment:
This seems like a regression -- shouldn't this still be distinct values?

Reply:
Good catch.